recipe-client-addon/lib/FilterExpressions.jsm

```diff
@@ -41,3 +43,32 @@ this.FilterExpressions = {
     return jexl.eval(onelineExpr, context);
   },
 };
+
+/**
+ * Return an array of the given object's own keys (specifically, its enumerable
+ * properties), or undefined if the argument isn't an object.
```

Comment:
No prob bob.

JEXL uses undefined as a fallback value for property access that fails in lieu of throwing a key error (`foo.bar.baz`, if `foo` is undefined or a number or something, returns undefined), and the kind of comparisons we'd expect will fail against undefined, like `(normandy.telemetry.main.iforgottherestofthepath.addons|keys).length > 0`, so I figured it's a good stand-in for cases where, in JavaScript, we'd throw.